File: 0102-binary-tree-level-order-traversal/0102-binary-tree-level-order-traversal.js
var levelOrder = function(root) {
     let q = [root], ans = []
    while (q[0]) {
        let qlen = q.length, row = []
        for (let i = 0; i < qlen; i++) {
            let curr = q.splice(0,1)
            row.push(curr[0].val)
            if (curr[0].left) q.push(curr[0].left)
            if (curr[0].right) q.push(curr[0].right)
        }
        ans.push(row)            
    }
    return ans
};

File: 0102-binary-tree-level-order-traversal/0102-binary-tree-level-order-traversal.py
class Solution {
public:
    vector<vector<int>> levelOrder(TreeNode* root) {
        vector<vector<int>>v;
        if(root ==NULL)return v;
        queue<TreeNode * >q;
        q.push(root);
        while(!q.empty()){
            vector<int>temp;
            int n=q.size();
            while(n--){
                TreeNode *t =q.front();
                temp.push_back(t->val);
                if(t->left)q.push(t->left);
                if(t->right)q.push(t->right);
                q.pop();
            }
            v.push_back(temp);
        }
        return v;
    }
};